packages/components/pages/f-contact-preferences/src/components/_tests/ContactPreferences.test.js

```diff
@@ -164,35 +163,57 @@ describe('ContactPreferences Component', () => {
 
         it('should set the `isFormDirty` flag to false', async () => {
             // Arrange
-            wrapper = mountContactPreferences();
+            wrapper = await mountContactPreferences();
             await wrapper.setData({ isFormDirty: true });
 
             // Act - re-mount the page to prove assertion
-            wrapper = mountContactPreferences();
+            wrapper = await mountContactPreferences();
 
             // Assert
             expect(wrapper.vm.isFormDirty).toEqual(false);
         });
 
-        it('should not call initialise method if the authorisation has not completed', () => {
+        it('should log an info log', async () => {
+            // Arrange & Act
+            wrapper = await mountContactPreferences();
+
+            // Assert
+            expect(logMocks.info).toHaveBeenCalledTimes(1);
```

Comment:
Might be best to check we are sending the correct tags (ditto for the other log tests).